reputation.php

```diff
@@ -559,6 +559,8 @@
 
 	$db->update_query("users", array('reputation' => (int)$reputation_value), "uid='{$uid}'");
 
+	$plugins->run_hooks("reputation_delete");
```

Comment:
Can we get it before `delete_query()` ? (L#551) Also, please set `$rid` instead of `$mybb->get_input('rid', MyBB::INPUT_INT)` @euantorano

Reply:
Done - @euantorano @Sama34